src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/bd562e0a-0208-4abe-a34d-284464234e99)

```suggestion
  src: url(./fonts/Avenir.ttc);
  font-weight: 300;
}

@font-face {
  font-family: Avenir-Book;
  src: url(./fonts/Avenir-Book.ttf);
  font-weight: 300;
```